internal/service/rds/cluster_test.go

```diff
@@ -2193,6 +2202,40 @@ func TestAccRDSCluster_enableHTTPEndpoint(t *testing.T) {
 	})
 }
 
+func TestAccRDSCluster_MigrateFromPluginSDK(t *testing.T) {
```

Comment:
Since this resource is so large, would it be worth writing a few different migration tests to cover the most common configurations?

Reply:
Good idea! We will definitely have to accept some diffs but we can definitely write some test to cover more basic configurations. Might be worth pointing out expected diffs in the changelog as well